draft-ietf-acme-acme.md

```diff
@@ -597,15 +592,15 @@ establish a new account with the server, prove control of an identifier, issue a
 certificate, and fetch an updated certificate some time after issuance.  The
 "->" is a mnemonic for a Location header pointing to a created resource.
 
-| Action               | Request        | Response     |
-|:---------------------|:---------------|:-------------|
-| Get a nonce          | HEAD new-nonce | 204          |
-| Register             | POST new-acct  | 201 -> acct  |
-| Submit an order      | POST new-order | 201 -> order |
-| Fetch challenges     | GET  authz     | 200          |
-| Respond to challenge | POST challenge | 200          |
-| Poll for status      | GET  order     | 200          |
-| Check for new cert   | GET  cert      | 200          |
+| Action               | Request           | Response        |
+|:---------------------|:------------------|:----------------|
+| Get a nonce          | HEAD new-nonce    | 204             |
+| Create account       | POST new-account  | 201 -> account  |
+| Submit an order      | POST new-order    | 201 -> order    |
+| Fetch challenges     | GET  authz        | 200             |
+| Respond to challenge | POST challenge    | 200             |
+| Poll for status      | GET  authz        | 200             |
```

Comment:
It seems like you should poll the order object here, because that's how you get the certificate URL.  Otherwise you have to go fetch the order once the authz turns good.